index.js

```diff
@@ -0,0 +1,10 @@
+delete process.env.BROWSER;
+
+if (! process.env.NODE_ENV ) {
+    throw new Error("You must specify a NODE_ENV of 'development' or 'production'");
+}
```

Comment:
In the IXA pallet standard this would be called `ENVIRONMENT` and have names like dev-local, staging, prod.